### ISSUE TITLE
Job failed with "Disconnected: Switch turned off"

### ISSUE THREAD

**Issue by 3Rafal** [ocurrent/current-bench#467]
http://autumn.ocamllabs.io:8081/job/2023-07-03/153313-ocluster-build-a15e1a

**Comment by punchagan** [comment]
We happened to restart the current-bench pipeline, after deploying some changes. The error message is unclear and could be confusing, but a [new build](https://autumn.ocamllabs.io/ocaml/merlin/pull/1640/base/master?worker=autumn&image=bench.Dockerfile) should automatically start for that commit once the pipeline restarts.

**Comment by 3Rafal** [comment]
Yup, I got confused and was wondering what happened and if the infrastructure is working correctly. Adding a message with an explanation would be a much better experience. :)
